build.gradle

```diff
@@ -328,6 +328,32 @@ project(':samza-sql') {
   }
 }
 
+project(':samza-tools') {
```

Comment:
The way i envison it, These are not just for samza sql tools.

e.g. generate-kafka-events can be used for any samza job. Say if a samza job produces events to event hubs. eh-consumer can be used to read events from event hubs. I could easily see generate-eh-events and generate-kinesis-events tools added to this tookit in future to aid testing and demo purposes.

Reply:
Agreed if the broader vision on this module is for testing and demo command line tools. But if that is the case, it would be nicer to use SystemFactory and configuration to load the producer/consumer in the test program and wrap them w/ specific system classes as generate-kafka-events.sh / generate-kinesis-events.sh, etc. Anyways, I don't think that is in the scope of Samza SQL tools and documentation. Please open a JIRA to track that.

Reply:
@nickpan47 Not sure whether it makes sense to add another layer of Samza config dependency on these tools. I feel this has potential to create spaghetti. And these tools are quite generic, if we decide to move these tools out of repo in future, We should be able to do them and not have Samza dependency on them. Anyway i will open a Jira to for future discussion around this.

Reply:
Created the jira here https://issues.apache.org/jira/browse/SAMZA-1533